### DISCUSSION TITLE
Privat Key Generator

### DISCUSSION THREAD

**Discussion by CePeHH** [BirthdayResearch/defichain-wallet#2843] [Ideas]
Its okay, the privatkey not to show in the lightwallet. But its possible public a generator with your "not standard" secure algorithm? 

So if people want to know they privat key, they get it from they seed.

Hope anyone understand my suggest and some people want an generator, too.

THX
CePe

**Comment by thedoublejay** [comment]
Hi @CePeHH, thanks for the suggestion. You can follow the progress with this issue - https://github.com/JellyfishSDK/jellyfish/issues/666
